app/controllers/main_controller.rb

```diff
@@ -1,4 +1,5 @@
 class MainController < ApplicationController
+  @user = User.all
```

Comment:
What are you trying to do there?

Reply:
I tried to create an instance variable but doing it here was a mistake